src/components/ListItem.js

```diff
@@ -1,37 +1,52 @@
 import React, { useState } from 'react';
 import { ITEMS, USERS } from '../constants';
 import { db } from '../lib/firebase';
-import { getDifferenceInHours } from '../lib/getDifferenceInHours';
+import { getDifferenceInHours, getDifferenceInDays } from '../lib/timeUtils';
+import calculateEstimate from '../lib/estimates';
+
 import '../CSS/ListItem.css';
 
 const ListItem = props => {
-  let hoursDiff = getDifferenceInHours(props.item.last_purchased);
+  const { item, token } = props;
   const [isPurchased, setPurchased] = useState(false);
+  let numberOfPurchases = item.number_purchases;
```

Comment:
```suggestion
  let numberOfPurchases = item.number_purchases || 0;
```
Might be good to have a fallback of 0 here, since there are some items in the database that don't have a `number_purchases` value.